src/Parts/Team.php

```diff
@@ -9,7 +9,7 @@ class Team
     public ?string $icon;
     public string $id;
     /**
-     * @var \Ragnarok\Fenrir\Parts\TeamMember[]
+     * @var TeamMember[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)